File: iosApp/iosApp/Screens/Profile/ProfileView.swift
//
//  ProfileView.swift
//  iosApp
//
//  Created by MacBook on 14.02.2024.
//  Copyright © 2024 orgName. All rights reserved.
//

import SwiftUI
import SharedSDK

struct ProfileView: View {
    
    let viewState:ProfileViewState
    let eventHandler: (ProfileEvent) -> Void
    
    var body: some View {
        
        let _ = print(viewState)
        
        Text("Profile")
    }
}

File: common/main/compose/src/androidMain/kotlin/profile/ProfileView.kt
package profile

import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import profile.models.ProfileEvent
import profile.models.ProfileViewState

@Composable
fun ProfileView(viewState: ProfileViewState, viewEvent: (ProfileEvent) -> Unit) {
    Text("Profile: $viewState")
}